src/sections/Community/Meshmates/index.js

```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import { Container, Row, Col } from "../../../reusecore/Layout";
+import Button from "../../../reusecore/Button";
+import ProfileCard from "../../../components/Profile-card";
+import MeshMatesWrapper from "./meshmates.style";
+import EmeritusGrid from "../Emeritus-grid";
+import { useStaticQuery, graphql } from "gatsby";
+
+import meshmate from "./meshmate.svg";
+import c_icon from "./c-icon.svg";
+import hero from "./hero.png";
+
+const Meshmates=() => {
+    const data=useStaticQuery(
+        graphql`
+            query meshmates {
+                allMdx(
+                    sort: { fields: [frontmatter___name], order: ASC }
+                    filter: { fields: { collection: { eq: "members" } }, frontmatter: { meshmate: { eq: "yes" } } }
+                ) {
+                    nodes {
+                        id
+                        frontmatter {
+                            name
+                            github
+                            twitter
+                            status
+                            linkedin
+                            location
+                            badges
+                            bio
+                            meshmate
+                            image_path{
+                                childImageSharp{
+                                    fluid(maxWidth: 200){
+                                        ...GatsbyImageSharpFluid
+                                    }
+                                }
+                                extension
+                                publicURL
+                            }
+                        }
+                        fields{
+                            slug
+                        }
+                    }
+                }
+            }
+        `
+    );
+    return (
+        <MeshMatesWrapper>
+            <div className="hero">
```

Comment:
Linting is wrong

Reply:
Its automated. I havent done any changes I think.

Reply:
This is due to change in the layer5ng branch eslint settings. Run `npm run lint` to fix it now